privacyidea/lib/caconnectors/localca.py

```diff
@@ -468,9 +468,13 @@ def get_templates(self):
         """
         content = {}
         if self.template_file:
-            with open(self.template_file, 'r') as content_file:
-                file_content = content_file.read()
-                content = yaml.safe_load(file_content)
+            try:
+                with open(self.template_file, 'r') as content_file:
+                    file_content = content_file.read()
+                    content = yaml.safe_load(file_content)
+            except (FileNotFoundError, PermissionError):
```

Comment:
I think you could use ``EnvironmentError`` instead and it would run on python 2 and 3.

Reply:
Thank's for the hint. That does the trick. EnvironmentError -> IOError, OSError, ...